pkgs/development/python-modules/fbs/default.nix

```
@@ -0,0 +1,47 @@
+{
+  lib,
+  python,
+  buildPythonPackage,
+  fetchFromGitHub,
+  unittestCheckHook,
+  rsa,
+}:
+
+buildPythonPackage rec {
+  pname = "fbs";
+  version = "1.2.7";
+
+  format = "setuptools";
```

```suggestion
  pyproject = true;
```
and use `build-system`.